static/app/views/issueDetails/streamline/issueTagsPreview.tsx

```diff
@@ -432,7 +438,7 @@ const VerticalIssueTagsButton = styled(LinkButton)`
   margin: ${space(1)} ${space(2)} ${space(1)} ${space(1)};
   padding: ${space(1)} ${space(1.5)};
   text-align: center;
-  width: 58px;
+  min-width: 55px;
   height: unset;
```

Comment:
```suggestion
  align-self: center;
```
<img width="1288" alt="SCR-20250331-jvud" src="https://github.com/user-attachments/assets/c0ace0fe-6032-478f-8c4a-13371d805751" />